### DISCUSSION TITLE
How to distribute milvus 2.X cluster across machines

### DISCUSSION THREAD

**Discussion by zhhhsdtc** [milvus-io/milvus#20032] [Q&A and General discussion]
I have seen docs of milvus 2.X with chapter Install Milvus -> Milvus Cluster，all of them don't recommend that deploy milvus cluster in product environments，does any docs to guide how to deploy

**Comment by martinenkoEduard** [comment]
I have the same question! The documentation really needs more information about distributed cluster installation!

**Comment by yhmo** [comment]
To deploy milvus cluster for production, helm chart is the recommended way:
https://milvus.io/docs/v2.1.x/install_cluster-helm.md#Install-Helm-Chart-for-Milvus
